docs-chef-io/content/server/upgrade_server_ha_v2.md

```diff
@@ -197,22 +399,22 @@ Upgrading to Chef Backend 3.0 require first upgrading Chef Backend to version 2.
     dpkg -i PATH_TO_FILE.deb
     ```
 
-3. Upgrade on follower nodes first:
+1. Upgrade on follower nodes first:
 
     ```bash
     chef-backend-ctl upgrade
     ```
 
-4. Upgrade the leader node with the `--failover` option:
+1. Upgrade the leader node with the `--failover` option:
 
     ```bash
     chef-backend-ctl upgrade --failover
     ```
 
-5. To continue the upgrades on Chef Infra Server frontend nodes using this back-end cluster, see [Upgrade Front-ends Associated with a Chef Backend Cluster]({{< relref "install_server_ha/#upgrading-chef-infra-server-on-the-frontend-machines" >}}).
+1. To continue the upgrades on Chef Infra Server front-end nodes using this back-end cluster, see [Upgrade Front-ends Associated with a Chef Backend Cluster]({{< relref "install_server_ha.md/#upgrading-chef-infra-server-on-the-frontend-machines" >}}).
 
 ## DRBD/Keepalived HA to Chef Backend 2.x
 
-DRBD configurations are [End of Life Products]({{< relref "versions#end-of-life-eol" >}}) and no longer supported.
+DRBD configurations are [End of Life Products]({{< relref "versions.md#end-of-life-eol" >}}) and no longer supported.
 
 For a guide to migrating to Chef Backend from DRBD see the [Best Practices for Migrating Your Chef Server](https://blog.chef.io/2018/04/06/best-practices-for-migrating-your-chef-server/) and the associated webinar from the [Chef Blog](https://blog.chef.io/).
```

Comment:
Heads up this blog post is no more. We should nuke that while we're in here

Reply:
good catch - looks like this is a new place it lives in: https://www.chef.io/blog/best-practices-for-migrating-your-chef-server